src/daemon/http/server.rs

```diff
@@ -498,13 +498,18 @@ pub async fn metrics(req: Request) -> RoutingResult {
                 roas_total: HashMap::new(),
             };
 
-            let mut parent_stats: HashMap<Handle, ParentStatuses> = HashMap::new();
+            let mut ca_parent_statuses: HashMap<Handle, ParentStatuses> = HashMap::new();
```

Comment:
If both hash maps are always updated together and walked one after the other does it make sense to combine them?

Reply:
separate sets